gradle-plugins/compose/src/test/kotlin/org/jetbrains/compose/test/tests/integration/GradlePluginTest.kt

```diff
@@ -106,7 +106,7 @@ class GradlePluginTest : GradlePluginTestBase() {
     fun testOldestKotlinJsMpp() = with(
         testProject(
             "application/jsMpp",
-            testEnvironment = defaultTestEnvironment.copy(kotlinVersion = oldestSupportedKotlinVersion)
+            testEnvironment = defaultTestEnvironment.copy(kotlinVersion = "2.2.0-RC2")
```

Comment:
Why it's not the constant anymore? Why it's RC and not stable version?
It will be tricky to find it during the next update